src/store.ts

```diff
@@ -157,6 +161,12 @@ const store: TypedVuexStore = new TypedVuexStore({
     setIsTeleportModalOpen(state: VuexStoreState, newTeleportModalValue: boolean) {
       state.isTeleportModalOpen = newTeleportModalValue;
     },
+    setPlans(state: VuexStoreState, newPlans: Plan[]) {
```

Comment:
```suggestion
    setPlans(state: VuexStoreState, newPlans: readonly Plan[]) {
```
This tells the type system that `setPlans` won't modify `newPlans`